packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/InviteCollaboratorForm.vue

```diff
@@ -289,17 +289,7 @@ export default defineComponent({
       )
       const { data: groupData } = yield client.groups.listGroups('displayName', null, `"${query}"`)
 
-      const users = (userData.value || []).map((u) => ({
-        ...u,
-        shareType: unref(resourceIsSpace) ? ShareTypes.spaceUser.value : ShareTypes.user.value
-      })) as CollaboratorAutoCompleteItem[]
-
-      const groups = (groupData.value || []).map((u) => ({
-        ...u,
-        shareType: unref(resourceIsSpace) ? ShareTypes.spaceGroup.value : ShareTypes.group.value
-      })) as CollaboratorAutoCompleteItem[]
-
-      autocompleteResults.value = [...users, ...groups]
+      autocompleteResults.value = [...userData.value, ...groupData.value]
```

Comment:
Fyi I simplified this a bit, there is no need to do a mapping before looping through the results here - we do one at the end anyways (see down below).